docs/pages/router/overview.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Expo Router
+description: Welcome to Expo Router
+sidebar_title: Overview
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon } from '@expo/styleguide-icons';
+
+Expo Router is a routing library for React Native applications built with Expo. It provides a way to manage navigation between screens in your app, allowing users to move between different parts of your app's UI seamlessly, using the same components on multiple platforms (iOS, Android, Web, etc).
+
+Expo Router brings the best routing concepts from the web to native iOS and Android apps. Every file in the **app** directory automatically becomes a route in your mobile navigation, making it easier than ever to build, maintain, and scale your project. It's built on top of our powerful [React Navigation suite](https://reactnavigation.org/) enabling truly native navigation. The entire deep linking system is automatically generated live, so users can share links to any route in your app.
+
+## Quick Start
+
+Run the following to create a project with `expo-router` setup:
+
+<Terminal
+  cmd={['$ npx create-expo-app@latest --example with-router']}
+  cmdCopy="npx create-expo-app@latest --example with-router"
+/>
+
+<BoxLink
+  title="Example source"
+  Icon={BookOpen02Icon}
+  description="See the example source code on GitHub."
+  href="https://github.com/expo/examples/tree/master/with-router"
+/>
```

Comment:
```suggestion
<BoxLink
  title="Example app"
  Icon={GithubIcon}
  description="See the source code for example app on GitHub."
  href="https://github.com/expo/examples/tree/master/with-router"
/>
```

Let's use `GithubIcon` in this scenario. We do that in other places as well where there is an example app/source code that points towards a GitHub repo. Example:
<img width="879" alt="CleanShot 2023-05-19 at 02 19 24@2x" src="https://github.com/expo/expo/assets/10234615/ba0bcb69-e0ae-4ff9-8dbd-394dbaf6ce15">